libcxx/include/__vector/vector.h

```diff
@@ -594,6 +598,30 @@ class _LIBCPP_TEMPLATE_VIS vector {
   _LIBCPP_CONSTEXPR_SINCE_CXX20 _LIBCPP_HIDE_FROM_ABI void
   __assign_with_size(_ForwardIterator __first, _Sentinel __last, difference_type __n);
 
+  template <class _Iterator,
+            __enable_if_t<!is_same<decltype(*std::declval<_Iterator&>())&&, value_type&&>::value, int> = 0>
+  _LIBCPP_CONSTEXPR_SINCE_CXX20 _LIBCPP_HIDE_FROM_ABI void
+  __insert_assign_n_unchecked(_Iterator __first, difference_type __n, pointer __position) {
+    for (pointer __end_position = __position + __n; __position != __end_position; (void)++__position, ++__first) {
```

Comment:
```suggestion
    for (pointer __end_position = __position + __n; __position != __end_position; ++__position, (void)++__first) {
```

I think it is more appropriate to guard against `operator,` here.

Reply:
Thanks! Although I believe the effects are exactly same.

It's conventional to `void`-cast the first operand in the standard wording. But in libc++ `void`-casting the second one seems conventional. What an unfortunate style mismatch.